index.js

```diff
@@ -64,7 +63,7 @@ function Bootprint (withData, targetDir) {
 function loadFromFileOrHttp (fileOrUrlOrData) {
   // If this is not a string,
   // it is probably already the raw data.
-  if (!_.isString(fileOrUrlOrData)) {
+  if (!typeof fileOrUrlOrData === 'string') {
```

Comment:
I think this lint-error may be gone if the expression line is `typeof fileOrUrlOrData !== 'string'`